vector/src/main/java/im/vector/adapters/VectorParticipantsAdapter.java

```diff
@@ -262,7 +262,8 @@ private void addContacts(List<ParticipantAdapterItem> list) {
                             if (DinsicUtils.isFromFrenchGov(email)) {
                                 findGovEmail = true;
                                 if (mUsedMemberUserIds != null && !mUsedMemberUserIds.contains(participant.mUserId)) {
-                                    list.add(participant);
+                                        DinsicUtils.removeParticipantIfExist(list,participant);
```

Comment:
1- Why don't you use here `DinsicUtils.participantAlreadyAdded`?
2- Why do you have to add this check here? did you observe some duplicates in the list?